tests/components/shopping_list/test_todo.py

```diff
@@ -457,7 +457,12 @@ async def test_move_item(
     summaries = [item["summary"] for item in items]
     assert summaries == ["item 1", "item 2", "item 3", "item 4"]
 
-    resp = await ws_move_item(uids[src_idx], dst_idx)
+    # Prepare items for moving
+    previous_uid: str | None = None
+    if dst_idx is not None:
+        previous_uid = uids[dst_idx]
```

Comment:
We don't know it until it is created since uids are created on the fly. We could seed with a fixed database, or mock out uid creation deep in the library, though my thought is that might be too tight of a coupling with an internal detail. However ical does this for it's own tests so there are some util hooks for that.

Reply:
(I was replying for ical though same techniques could be used across both)

Reply:
Hmm ok. I guess it's ok like this then.